app/controllers/books_controller.rb

```diff
@@ -0,0 +1,51 @@
+class BooksController < ApplicationController
+  layout 'authors_header'
+
+  def index
+    @books = Book.all
+  end
+
+  def show
+    @book = Book.find(params[:id])
```

Comment:
Create one method for finding a book and use the same method every where, using before_action callback.